File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/messagemgr/Message.java
package com.tencentcs.iotvideo.messagemgr;

import androidx.annotation.NonNull;

/* loaded from: classes2.dex */
public class Message {
    public int error;
    public long id;
    public int type;

    public Message(int type, long id, int error) {
        this.type = type;
        this.id = id;
        this.error = error;
    }

    @NonNull
    @Override
    public String toString() {
        return "type: " + this.type + ", id: " + this.id + ", error: " + this.error;
    }
}

File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/messagemgr/Message.kt
package com.tencentcs.iotvideo.messagemgr

open class Message(@JvmField var type: MessageType, @JvmField var id: Long, @JvmField var error: Int) {
    override fun toString(): String {
        return "type: " + this.type + ", id: " + this.id + ", error: " + this.error
    }
}
